src/esp/sim/Simulator.cpp

```diff
@@ -51,6 +51,20 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
     sceneFilename = cfg.scene.filepaths.at("mesh");
   }
 
+  // create pathfinder and load navmesh if available
```

Comment:
Some of this is now duplicated on [the python side](https://github.com/facebookresearch/habitat-sim/blob/6deb47b45e4e4548378d82f733aec2f8913e2c12/habitat_sim/simulator.py#L163). Could be a separate PR to refactor if everything is still functioning.